pkg/rancher-desktop/components/Preferences/VirtualMachineVolumes.vue

```diff
@@ -15,6 +15,10 @@ export default Vue.extend({
       type:     Object as PropType<Settings>,
       required: true,
     },
+    selectTab: {
+      type:     Function,
+      required: true,
+    },
```

Comment:
I think we should remove every instance of this prop and replace with emitting an event.